config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -308,15 +294,8 @@ periodics:
       - --env=ENABLE_POD_SECURITY_POLICY=true
       env:
       image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220908-70b61d242b-master
-      resources:
-        requests:
-          cpu: 1000m
-          memory: 3Gi
-        limits:
-          cpu: 1000m
-          memory: 3Gi
+      resources: *id001
```

Comment:
Should this `resources: *id001` still be here?

Reply:
Yes... as resources still need to be added here but with the correct indentation now.... the script abriviates the useage

Reply:
I believe `resources: *id001` was being added with wrong indentation before (2 lines below)

Reply:
Yes... as resources still need to be added here but with the correct indentation now.... the script abriviates the useage...I believe resources: *id001 was being added with wrong indentation before (2 lines below)

Reply:
resources: *id001 in the original script was intended to replace the template values but it got appended due to the bug in the script

Reply:
Yes, but now it is empty. If it needs to be empty, then we don't need the `resources: *id001` line. But I suspect that it is meant to have some valid content.

Reply:
it points to and populates from
```resources: &id001
        requests:
          cpu: 1000m
          memory: 3Gi
          cpu: 2000m
          memory: 6Gi```

Reply:
its short hand code using an anchor(&) and and alias (*)

Reply:
I learned something new today - I wasn't aware that YAML can do that.

But I still find it odd that a generated file uses this mechanism. I'll defer to the owner of this particular job whether they want this.

Reply:
where does it list specific job owners?